18_regexp_lab/01.py

```diff
@@ -9,4 +9,17 @@
 Write a program that takes a property file name and key
 as command line arguments and prints the requested value
 """
+import sys
+import re
 
+(_,filename,key) = sys.argv
+
+with open(filename, "r") as f:
+    for line in f:
+        if line[0] == '\n' or line[0] == '#': continue
+        keyvalue = re.search(r'^(\w+)[ ]*=[ ]*(\w+)$',line)
+        if keyvalue == None:
+            print "'{}' is not a key value line".format(line)
```

Comment:
Generally it's better to `continue` after this print, instead of writing most of the function code in the `elif` block